xml/System.Drawing.Printing/PrinterSettings.xml

```diff
@@ -1654,13 +1654,16 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets a value indicating whether this printer supports color printing.</summary>
+        <summary>On .NET Frameworks 1.1 to 4.8, .NET Core framework versions 3.1 and 3.0, and on .NET Framework Extensions 2.1 to 3.1, gets a value indicating whether this printer uses multiple adjacent color bits for each pixel. This value does not guarantee if this printer supports color.<br/>
+          On .NET 5, gets a value indicating whether this printer supports color printing.</summary>
```

Comment:
@gewarren is there a preferred way of wording? I think in the last couple of PRs you worded it something like this:
```suggestion
        <summary>In all .NET Framework and .NET Core versions, and in .NET Framework Extensions 2.1 to 3.1 gets a value indicating whether this printer uses multiple adjacent color bits for each pixel. This value does not guarantee if this printer supports color.<br/>
          Starting .NET 5 gets a value indicating whether this printer supports color printing.</summary>
```

Reply:
@RussKie I left a suggestion for how I would work it. I usually using the "Starting in version x..." verbiage in the breaking change docs, but not so much in the API ref docs.